data/json/items/comestibles/nuts.json

```diff
@@ -4,62 +4,56 @@
     "id": "pine_nuts",
     "name": "pine nuts",
     "name_plural": "pine nuts",
-    "weight": 28,
+    "weight": 30,
     "color": "brown",
     "spoils_in": 1080,
     "container": "bag_plastic",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": -2,
     "healthy": 1,
-    "calories": 43,
+    "calories": 202,
     "description": "Tasty crunchy nuts from a pinecone.",
     "price": 90,
     "material": "nut",
     "volume": 1,
     "flags": [ "EDIBLE_FROZEN" ],
-    "charges": 2,
+    "charges": 4,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.

In nuts.json, you change charges on lines (other than this one):
44, 80, 114, 137, 171, 194, 214, 247, 294, 328, 385, and 416

Reply:
not really sure a price change is necessary here, as the weights didn't really change much. only the volume changed here.